src/components/ExpensiPicker.js

```diff
@@ -27,27 +35,38 @@ class ExpensiPicker extends PureComponent {
 
     render() {
         const {
-            label, isDisabled, ...pickerProps
+            errorText,
+            hasError,
+            label,
+            isDisabled,
+            ...pickerProps
         } = this.props;
         return (
-            <View
-                style={[
-                    styles.expensiPickerContainer,
-                    this.state.isOpen && styles.borderColorFocus,
-                    isDisabled && styles.inputDisabled,
-                ]}
-            >
-                {label && (
+            <>
```

Comment:
Is this new outer view necessary?

Reply:
I think it _is_ because the error text needs to live outside of the first View component. But it would be great if you could confirm that.

Reply:
It is because of that. If I put it inside the existing view, the error text appears inside the borders of the input.